### PR TITLE
Security Scans: Exclude windows container scans

### PR DESCRIPTION
**Pull request by dsotirakis**
**What is this feature?**

Excludes windows container scans due to https://github.com/aquasecurity/trivy/issues/1392.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana-delivery/issues/156

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: scripts/drone/utils/windows_images.star [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""
+This module contains all the windows docker images that are used to build test and publish Grafana.
+All the windows images needed to be in a different file than the other images, since they cannot be scanned
```
Comment (zerok) [inline scripts/drone/utils/windows_images.star:3]: Would having a separate variable in images.star also worked?

File: scripts/drone/utils/windows_images.star [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+by trivy. Related issue: https://github.com/aquasecurity/trivy/issues/1392
+"""
+
+windows_images = {
+    "windows_image": "mcr.microsoft.com/windows:1809",
```
Comment (zerok) [inline scripts/drone/utils/windows_images.star:8]: The prefix `windows_` feels redundant here 🙂

**Review by zerok — COMMENTED** [review]
Some minor things. Lgtm otherwise 🙂

**Review by zerok — APPROVED** [review]
Some minor things. 👍 otherwise

File: scripts/drone/utils/windows_images.star (hunk 2 above)
Comment (dsotirakis) [reply scripts/drone/utils/windows_images.star:8]: Agreed!

**Pull request merged**